<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>781</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="two_split">
     <item>
      <widget class="QFrame" name="Left_Component_Frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="layoutWidget2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>131</width>
          <height>301</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>页面1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="function_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>功能</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="log_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>日志</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="setting_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>设置</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="Right_Display_Frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>5</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>-1</y>
          <width>641</width>
          <height>651</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="page">
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>103</x>
            <y>161</y>
            <width>281</width>
            <height>211</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>48</pointsize>
           </font>
          </property>
          <property name="text">
           <string>界面1</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="function_page">
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>271</width>
            <height>101</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>48</pointsize>
           </font>
          </property>
          <property name="text">
           <string>功能列表</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget3">
          <property name="geometry">
           <rect>
            <x>-10</x>
            <y>120</y>
            <width>651</width>
            <height>481</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="function_page_layout">
           <item>
            <widget class="QFrame" name="function_list">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QWidget" name="verticalLayoutWidget">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>0</y>
                <width>321</width>
                <height>481</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="functionListLayout">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="sizeConstraint">
                <enum>QLayout::SetMinAndMaxSize</enum>
               </property>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="function_display">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QStackedWidget" name="function_widget">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>301</width>
                <height>461</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <widget class="QWidget" name="defaut_page">
               <widget class="QLabel" name="function_title">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>0</y>
                  <width>71</width>
                  <height>41</height>
                 </rect>
                </property>
                <property name="text">
                 <string>说明</string>
                </property>
               </widget>
               <widget class="QLabel" name="function_describe">
                <property name="geometry">
                 <rect>
                  <x>20</x>
                  <y>50</y>
                  <width>261</width>
                  <height>41</height>
                 </rect>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </widget>
              <widget class="QWidget" name="page_3"/>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="log_page">
         <widget class="QPlainTextEdit" name="LogTextArea">
          <property name="geometry">
           <rect>
            <x>23</x>
            <y>60</y>
            <width>601</width>
            <height>571</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="setting_page">
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>90</y>
            <width>201</width>
            <height>491</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="setting_layout">
           <item>
            <widget class="QCheckBox" name="auto_start_checkbox">
             <property name="text">
              <string>是否开机自启</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="Button_Display_Frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>620</y>
      <width>781</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="version_label">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>0</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>v1.0.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="version_update_button">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>0</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>203</height>
      </size>
     </property>
     <property name="text">
      <string>检查升级</string>
     </property>
    </widget>
    <widget class="QPushButton" name="check_update_button">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>0</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>203</height>
      </size>
     </property>
     <property name="text">
      <string>检查升级</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
